server/src/controllers/StatisticsController.ts

```diff
@@ -0,0 +1,121 @@
+import { IItem } from "../definitions/IItem";
+import { Item } from "../definitions/Item";
+import { ItemScores } from "../ItemScores";
+
+export class MissingItems {
+  public missing: number = 0;
+  public score: number = 0;
+  public found: number = 0;
+
+  public constructor() {}
+}
+
+export class StatisticsController {
```

Comment:
it's not really a controller is it? it doesn't have API methods I think

Reply:
True 😄 
What shall I call it? StatisticsParser? DataHandler?
Let me know what you suggest and where to put it 👍

Reply:
something like `ItemScoreCalculator`? I'd just make a new folder called `utils` next to `controllers`(in the root) for now :)